Scripts/Mac/PublishGit.sh

```diff
@@ -0,0 +1,15 @@
+. "$(dirname ${BASH_SOURCE[0]})/InitializeEnvironment.sh"
+
+GITPATH=$1
+INSTALLER=$(basename $GITPATH)
+
+GITPUBLISH=$VFS_OUTPUTDIR/Git
+if [[ ! -d $GITPUBLISH ]] ; then
+  mkdir $GITPUBLISH
+fi
+
+find $GITPUBLISH -type f ! -name $INSTALLER -delete
```

Comment:
Yes, that's the intent. Since we're copying a versioned installer into a non-versioned directory, I want to ensure that we don't have multiple. The correctness of the installation during PrepFunctionalTests.sh depends on their being exactly the one dmg we want to install in $VFS_OUTPUTDIR/Git